File: Datasets/TrickyBugs/GenProgs/tc_generated_progs_cpp/p03803/p03803_num4_parsed.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n, m;
    cin >> n >> m;
    if(n == m)
        cout << "Draw";
    else if (n == 1)
        cout << "Alice";
    else if (m == 1)
            cout << "Bob";
    else if(n > m)
        cout << "Alice";
    else if(m > n)
        cout << "Bob";
}

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03803/p03803_num4_parsed.py
# Program

# Read the inputs
A, B = map(int, input().split())

# Check the outcome of the game
if A == 1:
    if B == 1:
        print("Draw")
    else:
        print("Alice")
elif B == 1:
    print("Bob")
elif A > B:
    print("Alice")
elif B > A:
    print("Bob")
else:
    print("Draw")